lms/static/js/student_account/views/RegisterView.js

```diff
@@ -238,6 +238,35 @@
                             });
                         };
 
+                    var checkbox_is_adg_employee = $('.checkbox-is_adg_employee');
+                    var select_company = $('.select-company');
+                    var register_company = $('#register-company');
```

Comment:
Yes it is kind of different. I need to mark the **select** for company as required so that is register-company and We have to change place of select-company. select-company is a **div** and it contains register-company**register-company**.